bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala

```diff
@@ -392,6 +393,9 @@ object CatalogAdapter {
 
     }
 
+    override def postProcessFieldDescriptor(fieldId: FieldId): FieldId =
+      if (fieldId.id == "DESIGNATION") fieldId.copy(id = fieldId.id.toLowerCase) else fieldId
```

Comment:
I guess this is the crux of the matter but I don't understand.  What happens if you just always `toLowerCase` "DESIGNATION"?

Reply:
you mean if you lower case all field.ids? That would be fine with Gaia as it only produces one ID. However other catalogs behave differently and a blanket change would break them (Simbad, ppxml)

In general if we wed'd read `ID` and if not found `name`. 
Previously we'd get
`<FIELD arraysize="*" datatype="char" name="designation" ucd="meta.id;meta.main">`

and now
`<FIELD ucd="meta.id;meta.main" name="designation" datatype="char" arraysize="*" ID="DESIGNATION">`

With the previous logic we'd parse this as a `FieldId("DESIGNATION")` and it wouldn't match with how we do the query which is with the field in lowercase.

An obvious solution would be to send the query with the field in uppercase but then it reverts to the previous answer and we'd fail to parse